AutoRest/AutoRest.Core/ClientModel/Method.cs

```diff
@@ -59,6 +59,27 @@ public Method()
         public List<Parameter> Parameters { get; private set; }
 
         /// <summary>
+        /// Gets the parameter groups.
+        /// </summary>
+        public IEnumerable<string> ParameterGroups
```

Comment:
This will be replaced with two "Mapping" properties - one for input and one for output.